services/frontend-service/src/ui/Pages/Locks/LocksPage.test.tsx

```diff
@@ -17,12 +17,12 @@ Copyright 2021 freiheit.com*/
 import { render } from '@testing-library/react';
 import { LocksPage } from './LocksPage';
 
-describe('EnvironmentsPage', () => {
+describe('LocksPage', () => {
     const getNode = (): JSX.Element | any => <LocksPage />;
     const getWrapper = () => render(getNode());
 
     it('Renders full app', () => {
         const { container } = getWrapper();
-        expect(container.textContent).toBe('List All Locks Here');
+        expect(container.getElementsByClassName('mdc-env-data-table')[0]).toBeTruthy();
```

Comment:
I think this test is useless maybe check for the two titles ' environment locks ' & ' application locks ' or maybe get rid of it

Reply:
I ended up maintaining the test but now checking both the titles